src/components/05_pages/Content/Content.js

```diff
@@ -80,221 +80,286 @@ class Content extends Component {
-            {this.props.contentList.map(
-              ({
-                type,
-                attributes: { changed, nid, status, title },
-                relationships,
-              }) => (
-                <TableRow key={nid}>
-                  <TableCell>{title}</TableCell>
-                  <TableCell>{this.props.contentTypes[type].name}</TableCell>
-                  <TableCell>
-                    {this.props.includes['user--user'][
-                      relationships.uid.data.id
-                    ] ? (
-                      <Link
-                        to={`/user/${
-                          this.props.includes['user--user'][
-                            relationships.uid.data.id
-                          ].attributes.uid
-                        }`}
-                      >
-                        {
-                          this.props.includes['user--user'][
-                            relationships.uid.data.id
-                          ].attributes.name
-                        }
-                      </Link>
-                    ) : (
-                      'Anonymous (not verified)'
-                    )}
-                  </TableCell>
-                  <TableCell>
-                    {(status && 'Published') || 'Unpublished'}
-                  </TableCell>
-                  <TableCell>
-                    {new Intl.DateTimeFormat(navigator.language, {
-                      weekday: 'long',
-                      year: 'numeric',
-                      month: 'long',
-                      day: 'numeric',
-                      hour: 'numeric',
-                      minute: 'numeric',
-                    }).format(new Date(changed * 1000))}
-                  </TableCell>
-                  <TableCell style={{ whiteSpace: 'nowrap' }}>
-                    <IconButton
-                      aria-label="edit"
-                      className={styles.button}
-                      component={Link}
-                      to={`/node/${nid}/edit`}
-                    >
-                      <EditIcon />
-                    </IconButton>
-                    <IconButton
-                      aria-label="delete"
-                      className={styles.button}
-                      component={Link}
-                      to={`/node/${nid}/delete`}
-                    >
-                      <DeleteIcon />
-                    </IconButton>
-                  </TableCell>
-                </TableRow>
-              ),
-            )}
-          </TableBody>
-        </Table>
-      </Paper>
-    </Fragment>
-  );
+                )}
+              </TableBody>
+            </Table>
+            <TablePagination
+              component="div"
+              count={count}
+              rowsPerPage={limit}
+              page={offset / limit}
+              onChangePage={this.pageChangeHandler}
+              rowsPerPageOptions={[limit]}
+              labelDisplayedRows={() => {}}
```

Comment:
What is the reason we don't show the current page we are on? i think this would be beneficial for the user. ❓

Reply:
Good suggestion! Done ✅